Makefile

```diff
@@ -40,34 +40,23 @@ clean-solver:
 install:
 	@ python utilities.py check-pydas
 	python setup.py install
-
-q2dtor:
-	@ echo -e "\nInstalling Q2DTor...\n"
-	@ echo -e "Q2DTor is a software for calculating the partition functions and themodynamic properties\
-	of molecular systems with two or more torsional modes developed by David Ferro Costas (david.ferro@usc.es)\
-	 and Antonio Fernandez Ramos (qf.ramos@usc.es) at the Universidade de Santiago de Compostela. Arkane can\
-	  integrate Q2DTor to compute the quantum mechanical partition function of 2D rotors.  \n\nFor use of Q2DTor\
- and HinderedRotor2D within Arkane please cite:  \n\nD. Ferro-Costas, M. N. D. S.Cordeiro, D. G. Truhlar, A.\
-		  Fernández-Ramos, Comput. Phys. Commun. 232, 190-205, 2018.\n"
-	@ read -p "Press ENTER to continue" dummy
-	@ git clone https://github.com/mjohnson541/Q2DTor.git external/Q2DTor --branch arkanepy3
 	
 decython:
 	# de-cythonize all but the 'minimal'. Helpful for debugging in "pure python" mode.
 	find . -name *.so ! \( -name _statmech.so -o -name quantity.so -o -regex '.*rmgpy/solver/.*' \) -exec rm -f '{}' \;
 	find . -name *.pyc -exec rm -f '{}' \;
 
 test-all:
-	nosetests --nocapture --nologcapture --all-modules --verbose --with-coverage --cover-inclusive --cover-erase --cover-html --cover-html-dir=testing/coverage --exe rmgpy arkane
+	python-jl -m pytest
 
 test test-unittests:
-	nosetests --nocapture --nologcapture --all-modules -A 'not functional' --verbose --with-coverage --cover-inclusive --cover-erase --cover-html --cover-html-dir=testing/coverage --exe rmgpy arkane
+	python-jl -m pytest -m "not functional not database"
```

Comment:
I think it needs to be `not functional and not database`. I get an error using pytest 7.1.2 but maybe it's different in other versions.

Reply:
Yes, pushing this now